infullmvp/src/main/java/com/infullmobile/android/infullmvp/InFullMvpFragment.java

```diff
@@ -32,14 +37,37 @@ public View onCreateView(@NonNull LayoutInflater inflater,
     @Override
     public void onActivityCreated(@Nullable final Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
-        Bundle bundle = getActivity().getIntent().getExtras();
+        if (!checkAllKeysAreUnique()) {
+            throw new IllegalStateException("Bundles cannot both have an extra with the same key");
```

Comment:
Bundle for your fragment cannot have the same keys as the one from your activity